recipe/meta.yaml

```
@@ -71,7 +71,7 @@ outputs:
         - openssl {{ openssl }}
         - zlib {{ zlib }}
       run:
-        - openssl 3.*
+        - openssl {{ openssl }}
```

```suggestion
        - openssl  # exact pin handled through openssl run_exports
```
`openssl {{ openssl }}` would create a whole bunch of conflicts all over the place since it would render to `openssl 3.0`.